include/mbedtls/mbedtls_config.h

```diff
@@ -3867,6 +3887,19 @@
  */
 //#define MBEDTLS_PSA_KEY_SLOT_COUNT 32
 
+/**
+ * \def MBEDTLS_PSA_STATIC_KEY_SLOT_BUFFER_SIZE
+ *
+ * Define the size (in bytes) of each static key buffer when
+ * #MBEDTLS_PSA_STATIC_KEY_SLOTS is set. If not
+ * explicitly defined then it's automatically guessed from available PSA keys
```

Comment:
Rather than `guessed`, would `determined` be better? `Guess` doesn't really have the right connotations

Reply:
I think guess is fine, given that we're using heuristics that we don't want to commit to.